### PR TITLE
Add a check for docker version to push fat manifest images

### PR DESCRIPTION
**Pull request by mkumatag**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #
This is for adding a check to avoid any corrupted fat manifest creation.

**Special notes for your reviewer**:
@dims @luxas 
**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by dims — APPROVED** [review]
/lgtm

File: test/images/image-util.sh [hunk 1]
```diff
@@ -92,8 +92,17 @@ build() {
   done
 }
 
+docker_version_check() {
+  docker_version=$(docker version --format '{{.Client.Version}}' | cut -d"-" -f1)
+  if [[ $docker_version != 18.06.0 && $docker_version < 18.06.0 ]]; then
```
Comment (ixdy) [inline test/images/image-util.sh]: is this a string-wise comparison? will it do the right thing when 18.09.00 or whatever is released?

(also, minor style nits: prefer quoting variables and using `${var}` over `$var`. also don't need the semicolons ending the next two lines.)

Comment (mkumatag) [reply test/images/image-util.sh]: @ixdy yes it is string comparison, I tested with all possible combination, things are fine. here is the sample run result - https://gist.github.com/mkumatag/4003fc248e99a7a190a9e3a15a54a6a5

**Comment by mkumatag** [comment]
@ixdy I just fixed those minor nits, ptal.

**Pull request merged**